src/api-binder.ts

```diff
@@ -283,9 +283,9 @@ export class APIBinder {
 		}
 
 		try {
-			const res = (await this.balenaApi
-				.get(reqOpts)
-				.timeout(timeout)) as Device[];
+			const res = (await Bluebird.resolve(this.balenaApi.get(reqOpts)).timeout(
```

Comment:
One of the breaking changes is using native promises and async/await in pinejs-client-request now, hence this wrapping for the `.timeout`